test/validation/api/ipsec/ipsec.c

```diff
@@ -269,10 +269,8 @@ int ipsec_check(odp_bool_t ah,
 	}
 
 	num = odp_ipsec_auth_capability(auth, auth_capa, MAX_ALG_CAPA);
-	if (num <= 0) {
-		fprintf(stderr, "Wrong auth capabilities\n");
+	if (num <= 0)
```

Comment:
I think this should result in validation test failure instead of just inactivation of the test. At this point ODP has already claimed to support the algorithm when queried through odp_ipsec_capability() and now when the test code wants to know more, ODP basically says that it does not support any key length with the algorithm.

This just cannot be fixed without fixing the CCM implementation problem that I discussed earlier or the validation tests would start failing (which would be right, but break the CI for everybody). I would leave this alone for now as it is not really related to the code changes in this patch.

Reply:
Could you remove this change?

I am going to soon send a patch to fix the capability issue and will put a CU_ASSERT in this location to make sure an implementation does not shy away from supporting an algorithm that it already advertised in capabilities.

Reply:
Sure.